test/PushNotificationTests/UnpackagedTests.cpp

```diff
@@ -95,7 +95,7 @@ namespace Test::PushNotifications
 
     void UnpackagedTests::VerifyUnregisterFails()
     {
-        if (!m_isSelfContained)
+        if (PushNotificationManager::Default().IsSupported())
```

Comment:
This doesn't look right. If the feature is unsupported, we get an E_UNEXPECTED error?

Reply:
E_UNEXPECTED is what we are throwing when a user alls Unregister without Register.

Reply:
I see.

Reply:
The test still calls Register, it just doesn't expect any API to throw in Unpackaged + SelfContained environment. I can add comments saying "PushNotificationManager APIs called in a SelfContained environment will not throw".